### PR TITLE
Add extra scenario for circuit breaker

### PR DESCRIPTION
**Pull request by Tomcli**
Add extra scenario to show how circuit breaker eject broken pods out of the load balancing pool.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -312,6 +312,35 @@ Now point your browser to:  `http://<IP:NodePort>`, enable your **developer mode
 
 > Note: using fault injection or mixer rule won't able to trigger the circuit breaker because all the traffic will be aborted/delayed before it get sent to the cloudant's Envoy.
 
+Next, let the circuit breaker eject any broken pod in you load balancing pool to avoid further failures. First, create a new cloudant database, cloudant-db2, that listens to the wrong host.
```
Comment (animeshsingh) [inline README.md]: what is a pool in the kubenretes conext ?  then we are going to add the two database instances in one pool. each instance is a represented by a pod of two containers (database container, sidecar container)

**Comment by animeshsingh** [comment]
@AnthonyAmanse @szihai please do a test run for this scenario once

**Comment by animeshsingh** [comment]
Merging it for now.

**Comment by szihai** [comment]
I got an error applying the timeout rule:
Error: 2 errors occurred:

* route rule must have a name
* route rule name must be a host name label

**Comment by Tomcli** [comment]
@szihai are you using istio 1.6?

**Comment by szihai** [comment]
Right, I'm using the latest version instead of 1.6 release.  Just want to put a note that this will not be working in the future version.

File: README.md [hunk 2]
```diff
@@ -312,6 +312,35 @@ Now point your browser to:  `http://<IP:NodePort>`, enable your **developer mode
...
+```shell
+kubectl get pods # check your cloudant-db-second name
+kubectl logs cloudant-db-second-xxxxxxx-xxxxx proxy --tail=150 # You can replace 150 with the number of logs you like to display.
+```
+As you can see, there will only be one HTTP call within the logs.
+
+Before you move to the next step, please remove the broken cloudant and circuit breaker policy.
+```shell
+kubectl delete -f manifests/deploy-broken-cloudant.yaml
+istioctl delete -f maniests/circuit-breaker-db.yaml
```
Comment (AnthonyAmanse) [inline README.md:341]: Typo here

**Comment by AnthonyAmanse** [comment]
The extra scenario works as intended. LGTM

**Pull request merged**